### DISCUSSION TITLE
alternative to airtable?

### DISCUSSION THREAD

**Discussion by ghost** [keeweb/keeweb#2176] [Q&A / Support] [answered]

**Comment by Kehino** [comment, marked as answer]
That seems suboptimal to me: you cannot script it, you can only reference between fields, and you probably don't need the file to be encrypted for your usecase.

You can have a look at https://www.getgrist.com/ or Nextcloud Tables (https://apps.nextcloud.com/apps/tables) which are two mature AirTable drop-in replacements I know of. You can explore other alternatives on https://alternativeto.net/software/airtable/
